monitoring/monitoring.go

```diff
@@ -117,17 +116,7 @@ func (l *listener) OnRunnableChange(started []e2e.Runnable) error {
 }
 
 type opt struct {
-	currentProcessAsContainer bool
-	scrapeInterval            time.Duration
-}
-
-// WithCurrentProcessAsContainer makes Start put current process PID into cgroups and organize
-// them in a way that makes cadvisor to watch those as it would be any other container.
-// NOTE: This option requires a manual on-off per machine/restart setup that will be printed on first start (permissions).
-func WithCurrentProcessAsContainer() func(*opt) {
-	return func(o *opt) {
-		o.currentProcessAsContainer = true
-	}
+	scrapeInterval time.Duration
```

Comment:
I guess we strictly do not need `opt` struct, since we can only set scrape interval

Reply:
Well, we have to, otherwise we will make breaking change which we can, but ideally we don't 🙃